corpus/headline.txt

```diff
@@ -14,11 +14,12 @@ Text
     (word)
     (tag
       (word)))
-  (line
-    (word)))
+  (block_end
+    (line_noeol
+      (word))))
 
 ================================================================================
-Headline with tag at the start
+headline with tag at the start
```

Comment:
use lowercase descriptions so that `tree-sitter test --filter headline` is easy to use.